src/containers/About/About.js

```diff
@@ -83,6 +83,11 @@ export /* istanbul ignore next */ class About extends Component {
   async fetchDashboardInfo() {
     try {
       const dashboardInfo = await getInstallProperties();
+      if (
```

Comment:
if we add other 'optional' properties in future this code will quickly get out of hand

what about the approach I shared on Slack last week? It would allow us to add new properties with minimal effort, and differentiate between those that are 'required' (i.e. should trigger an error notification) vs. optional.

![image](https://user-images.githubusercontent.com/2829095/76239565-b3538d80-6229-11ea-9a22-15c45cf39831.png)

Reply:
I did try that last week after you sent it via Slack but I couldn't get it to work 

If a optional property is installed and does somehow return an empty string as the version then that should probably be displayed as it might indicate an error somewhere in dashboard
The two bottom properties should be required anyway shouldn't they? As they are true or false and the ReadOnly impacts what gets displayed